index7.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&family=Raleway:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style7.css">
+    <title>hw7</title>
+</head>
+<body>
+    <header>
+        <h2>WHAT OUR CUSTOMERS ARE SAYING</h2>
+        <hr>
+        <h5>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h5>
+    </header>
+    <main class="main">
+        <div class="main__txt">
+            <h5 class="main__txt_h">Lorem ipsum dolor sit amet, 
```

Comment:
Block Element Modify 
class main__txt_h is not correct 
you should named like main__title